src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -1439,70 +1428,17 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
 
   private handleStripeError(
     error: StripeError | {message: string, code: string, decline_code?: string} | undefined,
-    context: string,
+    context: 'method_setup'| 'card_change'| 'confirm',
   ) {
     this.submitting = false;
-    this.stripeError = this.getStripeFriendlyError(error, context);
+    this.stripeError = getStripeFriendlyError(error, context);
     this.toast.showError(this.stripeError);
     this.stripeResponseErrorCode = error?.code;
 
     this.jumpToStep('Payment details');
     this.goToFirstVisibleError();
   }
 
```

Comment:
function below moved to stripe.service.ts to allow sharing with new component. Not sure if that's the best place to move it to - not sure if its idiomatic or not to put extra stuff in the file alongside the service class in Angular.